app/models/attendance.py

```diff
@@ -23,14 +25,16 @@ class LeaveStatus(models.TextChoices):
         Employee, on_delete=models.CASCADE, related_name="leaves"
     )
     updated_by = models.ForeignKey(
-        Employee,
+        AUTH_USER_MODEL,
         on_delete=models.CASCADE,
         related_name="managed_leaves",
         null=True,
         blank=True,
     )
     leave_from = models.DateField()
     leave_to = models.DateField()
+    description = models.TextField()
+    hr_comment = models.TextField(blank=True, null=True)
```

Comment:
only HR can add the comment here? if so then only HR can update this model?

Reply:
yes only hr